src/components/FilePreview/FilePreview.scss

```diff
@@ -0,0 +1,145 @@
+@use 'sass:math';
+@use '../variables';
+
+$block: '.#{variables.$ns}file-preview';
+
+$smallRoundedButtonSize: 24px;
+
+#{$block} {
+    --gc-_-box-shadow: none;
```

Comment:
no need in `gc` prefix these vars are local anyway. Let's follow the pattern `--_--{name}`, so it will be `--_--box-shadow`